doc/source/release/1.15.0-notes.rst

```diff
@@ -291,6 +292,17 @@ and support several Array API compatible array libraries in addition to NumPy
 
 - `scipy.stats.qmc.PoissonDisk` now accepts lower and upper bounds
   parameters ``l_bounds`` and ``u_bounds``.
+- `scipy.stats.fisher_exact` now supports two-dimensional tables with shapes
+  other than ``(2, 2)``.
+- `scipy.stats.truncate` adds the ability to truncate the distribution
+  underlying a random variable and `scipy.stats.abs` provides the ability to
+  take the absolute value of a random variable. The new ``FoldedDistribution``
```

Comment:
I don't think the `FoldedDistribution` is exposed in `scipy.stats`, or am I wrong here, @mdhaber ?

Reply:
You're right, thanks! I will read through this again now.